### PR TITLE
Proposal to add the floatThead plugin for JQuery

### PR DESCRIPTION
**Pull request by PhilippWendler**
Proposal to add the floatThead plugin for JQuery.

The author supports this proposal (cf. https://github.com/mkoryak/floatThead/issues/236).

### CODE REVIEW

**Comment by mkoryak** [comment]
yep, thanks for making this PR

File: files/jquery.floatthead/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "jquery.floatthead",
+  "repo": "mkoryak/floatThead",
+  "files": {
+    "include": ["dist/*.js"]
```
Comment (jimaek) [inline files/jquery.floatthead/update.json]: Use this instead `"dist/*.min.js`

**Comment by PhilippWendler** [comment]
I have updated the PR as you requested.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**